src/main/java/teammates/logic/api/Logic.java

```diff
@@ -1022,6 +1022,17 @@ public void deleteStudent(String courseId, String studentEmail) {
         studentsLogic.deleteStudentCascade(courseId, studentEmail);
     }
 
+    /**
+     * Deletes all the students in the course.
+     *
+     * @param courseId course id for the students
+     */
+    public void deleteAllStudentsInCourse(String courseId) {
+
+        Assumption.assertNotNull(courseId);
+        studentsLogic.deleteAllStudentsInCourse(courseId);
```

Comment:
Seems like it doesn't do the cascade though (delete feedback responses + delete student from respondents list) 😅 Currently it's only used in another method that deletes the entire course.